en/docs/learn/user-regisrtation-with-email-verification.md

```diff
@@ -0,0 +1,109 @@
+
+Let's learn how it's done!  
+
+## Set up  
+
+Follow the steps below to configure WSO2 Identity Server to enable password entry for account confirmation.
+
+
+1.	Open the `deployment.toml` file in the `<IS_HOME>/repository/conf` directory.
+
+	1.	To request email verification from the users who are registered with a default password, add the following configurations. 
+
+		```toml
+		[identity_mgt.user_onboarding]
+		enable_email_verification = true 
+		lock_on_creation=true
+		```
+
+	2.	To configure the email server to send emails requesting passwor entry, add the following configurations.
+
+		-	**from_address**: This is the email address from which the confirmation email will be sent.
+		-	**username**: This is the user name of the given email address.
+		-	**password**: This is the password of the given email address. 
+
+		```toml tab="Format"
+		[output_adapter.email]
+		from_address= ""
+		username= ""
+		password= ""
+		hostname= "smtp.gmail.com"
+		port= 587
+		enable_start_tls= true
+		enable_authentication= true
+		```
+
+		```toml tab="Sample"
+		[output_adapter.email]
+		from_address= "wso2iamtest@gmail.com"
+		username= "wso2iamtest"
+		password= "Wso2@iam70"
+		hostname= "smtp.gmail.com"
+		port= 587
+		enable_start_tls= true
+		enable_authentication= true
+		```
+
+		!!! warning "If you are using a Google email account"
+
+			Google has restricted third-party applications and less secure applications from sending emails by default. As WSO2 Identity Server acts as a third-party application when sending emails for password entry, follow the steps below to enable your Google email account to provide access to third-party applications.
+
+			1.	Access [https://myaccount.google.com/security](https://myaccount.google.com/security).
+
+			2.	Under **Signing in to Google** section, turn off the **2-step Verification** option. 
+
+
+2.	[Restart WSO2 Identity Server]((../../setup/running-the-product/)).
+
+3.	Sign in to the WSO2 Identity Server [Management Console](../../setup/getting-started-with-the-management-console/) at `https://<SERVER_HOST>:9443/carbon` as an administrator. 	
+
+4.	On the **Main** menu of the Management Console, click **Identity > Identity Providers > Resident**.
+
+	<img src="../../assets/img/learn/resident-menu-item.png" alt="Resident menu-item" width="200" style="border:1px solid grey">  
+
+5.	Under the **Account Management Policies** section, click **User Onboarding**.
+
+	<img src="../../assets/img/learn/user-onboarding-option.png" alt="User Onboarding Option" width="700" style="border:1px solid grey">  	
+
+6.	Select the **Enable User Email Verification** check box.
+
+	<img src="../../assets/img/learn/ask-password-enable-email-verification.png" alt="Ask password code expiry time text box" width="700" style="border:1px solid grey">  	  
+
+7.	You may enter the confirmation email validity period (in minutes) in the **Email verification code expiry time** text box.
```

Comment:
Shall we start this with "Enter the confirmation..." like the other instructions?

Reply:
This is an optional step. Optional steps should start with, 'You may'.